### ISSUE TITLE
Bundling OSC commands.

### ISSUE THREAD

**Issue by Dannington** [monsieurgustav/UE4-OSC#20]
Hi M.G.  As we've chatted about on the UE forums, it'd be great to be able to bundle a whole batch of OSC commands for situations where a lot of individual updates are required - hopefully this would help with efficiency in this case.

Dan

**Comment by davidmw** [comment]
Agreed. I struggled with very frequent freezes of UE4 during testing. No errors thrown but I had to force quit UE4 each time. Eventually started reducing the number of commands sent back to iPad. I had been doing brute force updates of all OSC controls on four tabs worth of UI - so was blasting 30+ commands as quickly as possible. The freeze disappeared when I got smarter about the update but it did seem like I had a problem with the command buffer or thread safety when sending OSC commands rapidly.
